lib/task/import/csvImportBaseTask.class.php

```diff
@@ -94,4 +94,185 @@ protected function acceptsOption($name)
+                ? $import->rowStatusVars[$column][$index]
+                : $import->rowStatusVars[$column][0];
+
+            // allow column value(s) to set event property
+            if (isset($definition['property']))
+            {
+              $eventData[($definition['property'])] = $value;
+            }
+
+            // allow column values(s) to set variable
+            if (isset($definition['variable']))
+            {
+              $$definition['variable'] = $value;
+            }
+          } else if (isset($definition['requiredError'])) {
+            throw new sfException('You have populated the eventActors column but not the eventTypes column.');
+          }
+        }
+
+        // if an event type has been specified, attempt to create the event
+        if ($eventType)
+        {
+          // do lookup of type ID
+          $typeTerm = $import->createOrFetchTerm(QubitTaxonomy::EVENT_TYPE_ID, $eventType);
+          $eventTypeId = $typeTerm->id;
+
+          // create event
+          $event = $import->createOrUpdateEvent($eventTypeId, $eventData);
+
+          // create a place term if specified
+          if ($place)
+          {
+            // create place
+            $placeTerm = $import->createTerm(QubitTaxonomy::PLACE_ID, $place);
+            $import->createObjectTermRelation($event->id, $placeTerm->id);
+          }
+        } else {
+          throw new sfException('eventTypes column need to be populated.');
+        }
+      }
+    }
+  }
+
+  /**
+   * Import creation events
+   */
+  static function importCreationEvents(&$import)
+  {
+    // copy legacy column data, if present, for backwards compatibility
+    $legacyColumns = array(
+      'creatorDates'      => 'creationDates',
+      'creatorDatesStart' => 'creationDatesStart',
+      'creatorDatesEnd'   => 'creationDatesEnd',
+      'creatorDateNotes'  => 'creationDateNotes'
+    );
+
+    foreach ($legacyColumns as $legacyColumn => $newColumn)
+    {
+      if (
+        isset($import->rowStatusVars[$legacyColumn])
+        && !isset($import->rowStatusVars[$newColumn])
+      )
+      {
+        $import->rowStatusVars[$newColumn] = $import->rowStatusVars[$legacyColumn];
+      }
+    }
+
+    // add creators and creation events
+    $createEvents = array();
+    if (isset($import->rowStatusVars['creators'])
+      && count($import->rowStatusVars['creators']))
+    {
+      foreach($import->rowStatusVars['creators'] as $index => $creator)
+      {
+        // Init eventData array and add creator name
+        $eventData = array('actorName' => $creator);
+
+        setupEventDateData($import, $eventData, $index);
+
+        // Add creator history if specified
+        if(isset($import->rowStatusVars['creatorHistories'][$index]))
+        {
+          $eventData['actorHistory'] = $import->rowStatusVars['creatorHistories'][$index];
+        }
+
+        array_push($createEvents, $eventData);
+      }
+    }
+    else if(
```

Comment:
Minor style nitpick:
Please put spaces after your ifs and foreach loops. Also I'm not really sure why this if condition spans 4 lines, you could fit it in 2. There are several cases of this in the commit.